robowflex_library/include/robowflex_library/builder.h

```diff
@@ -196,15 +214,15 @@ namespace robowflex
                                const Eigen::Quaterniond &orientation, const Eigen::Vector3d &tolerances,
                                const RobotPose &offset, const Eigen::Vector3d &axis, unsigned int n);
 
-        /** \brief Adds a set of regions to grasp a cylinder from the side. This function assumes the X-axis
-         * of the end-effector frame \a ee_name points "forward" for grasping.
-         *  \param[in] ee_name The name of the end-effector link.
-         *  \param[in] base_name The name of the frame of reference of \a pose.
-         *  \param[in] pose The pose of the frame to be rotated about.
-         *  \param[in] cylinder The cylinder to grasp.
-         *  \param[in] distance The distance from the cylinder to place the regions.
-         *  \param[in] depth The depth of boxes to create.
-         *  \param[in] n The number of regions to create.
+        /** \brief Adds a set of regions to grasp a cylinder from the side. This function assumes the
```

Comment:
These diffs are from my clang formatter. I'm using the lab clang-format file, it just didn't like the long lines.